tests/unit/info.tcl

```diff
@@ -334,21 +334,25 @@ start_server {tags {"info" "external:skip" "debug_defrag:skip"}} {
 
         test {stats: instantaneous metrics} {
             r config resetstat
+            r config set hz 100
             set retries 0
             for {set retries 1} {$retries < 4} {incr retries} {
-                after 1600 ;# hz is 10, wait for 16 cron tick so that sample array is fulfilled
+                after 1600 ;# Wait for 16 cron tick so that sample array is fulfilled
                 set value [s instantaneous_eventloop_cycles_per_sec]
                 if {$value > 0} break
             }
 
             assert_lessthan $retries 4
             if {$::verbose} { puts "instantaneous metrics instantaneous_eventloop_cycles_per_sec: $value" }
             assert_morethan $value 0
-            assert_lessthan $value [expr $retries*15] ;# default hz is 10
+            # Hz is configured to 100, so we expect a value of about 100, but in practice it will be lower
+            # because of imprecision in kernel wakeups, but we also have some other wakeups like clients cron.
+            assert_lessthan $value 100
```

Comment:
If we expect *about 100*, shouldn't we accept some range above and below 100?

    assert {$value >= 80 and $value <= 120}

Reply:
I was looking at this test.  It wasn't clear to me why we're expecting `value` to be less than `retries * 15`.

Retries should normally be 1.  It's not clear to me that it would ever go to 2.

So why, when we wait for 1600 ms, do we expect the 100ms cron to fire less than 15 times?

Reply:
I have no idea why there was a retry counter here, or why retry changed the expected value (theyre both instantaneous values).